docs-ui/stories/components/timeSince.stories.js

```diff
@@ -0,0 +1,115 @@
+import TimeSince from 'sentry/components/timeSince';
+import Tooltip from 'sentry/components/tooltip';
+
+export default {
+  title: 'Components/Time Since',
+  component: Tooltip,
+};
+
+export const _TimeSince = args => {
+  return <TimeSince {...args} />;
+};
+
+_TimeSince.args = {
+  date: new Date('Oct 31, 2020 9:00:00 PM UTC'),
+  disabledAbsoluteTooltip: false,
+  extraShort: false,
+  shorten: false,
+  suffix: '',
+  tooltipShowSeconds: false,
+  tooltipTitle: null,
+  tooltipUnderlineColor: 'black',
+};
+_TimeSince.argTypes = {
+  date: {
+    control: 'date',
+  },
+  tooltipUnderlineColor: {
+    control: 'select',
+    options: [
```

Comment:
I wonder if we can derive this list from our theme files somehow?

Reply:
I can get `Object.keys(lightColors)` from the theme.tsx file, but there's no existing export to get the list of aliases from what I can see. 

I could also remove the list because it is long and annoying... i feel like if it wasn't here then it'd be harder to spot during review.

Reply:
I suppose it's fine for now, maybe we'll find a better solution once we have more components that accept a color